hr_public_holidays/wizards/public_holidays_next_year_wizard.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Trobz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, models, fields
+
+
+_logger = logging.getLogger(__name__)
+
+
+class PublicHolidaysNextYearWizard(models.TransientModel):
+    _name = 'public.holidays.next.year.wizard'
+    _description = 'Creates public holidays for the year following the last'
+
+    @api.multi
+    def create_next_year_public_holidays(self):
+
+        ph_env = self.env['hr.holidays.public']
+        pholidays = ph_env.search([])
```

Comment:
An habit, it will make it easier to use later if necessary. In the next commit, it will also simplify a bit to avoid code on 2 lines for this:
```
        ph_env = self.env['hr.holidays.public']
        pholidays = self.template_ids or ph_env.search([])
```
If required, I can change, let me know.